### PR TITLE
[BIT] 引擎 dtype 转换能力增强；astype arange argmax Tensor.argmax full full_like logcumsumexp

### PR DESCRIPTION
**Pull request by Cutelemon6**
### dtype 转化能力增强
对于 api_config 中含有 dtype 的关键字参数，提供从 paddle.dtype 和 paddle.base.libpaddle.VarDesc.VarType 以及用 str 描述的 dtype 转换到 torch.dtype 的能力，具体见 base.py -- convert_dtype_to_torch_type()，对于 float32 的例子：

```python
if dtype in ["float32", numpy.float32, paddle.float32, paddle.base.libpaddle.VarDesc.VarType.FP32]:
    return torch.float32
```

此转换能力增强可降低以下 api 的修复难度，以下 api 未在之前的表格中出现：

```python
paddle.Tensor.argmax 
paddle.argmax 
paddle.full 
paddle.full_like 
paddle.linspace 
paddle.logcumsumexp 
paddle.logspace 
paddle.nn.functional.log_softmax 
paddle.nn.functional.sequence_mask 
paddle.nn.functional.softmax 
paddle.ones 
paddle.nn.quant.weight_only_linear 
paddle.ones_like 
paddle.sum 
paddle.zeros 
paddle.zeros_like 
paddle.empty 
```

### paddle.Tensor.astype
\_tmp_tensor_ = args[0], result = \_tmp_tensor_.to(dtype=dtype)
配置均通过，移入 support2torch_6.txt

### paddle.arange
result = torch.arange(start=start, end=end, step=step, dtype=dtype)
含有 Tensor 的配置有初始化问题，等待 @cszdrg 修复初始化问题，其它配置均通过，移入 support2torch_6.txt

### paddle.Tensor.argmax
\_tmp_tensor_ = args[0], result = \_tmp_tensor_.argmax(dim=axis, keepdim=keepdim)
配置均通过，移入 support2torch_6.txt

### paddle.argmax
result = torch.argmax(input=x, dim=axis, keepdim=keepdim)
配置均通过，移入 support2torch_6.txt

### paddle.full
result = torch.full(size=convert_to_list(shape), fill_value=convert_to_scalar(fill_value), dtype=dtype)

该配置有一条 OOM 的罕见报错，提示需要分配 >10000G 显存，未移出 merged 文件
`paddle.full(shape=Tensor([5],"int64"), fill_value=-2, )`

其它配置均通过，移入 support2torch_6.txt

### paddle.full_like

result = torch.full_like(input=x, fill_value=torch_kwargs['fill_value'], dtype=dtype)
配置均通过，移入 support2torch_6.txt

### paddle.logcumsumexp

```python
if axis is None:
    x_flat = x.flatten()
    result = torch.logcumsumexp(x_flat, dim=0)
else:
    result = torch.logcumsumexp(x, dim=axis)
```
配置均通过，移入 support2torch_6.txt

### CODE REVIEW

File: tester/paddle_to_torch/rules.py
Comment (mzj104) [inline tester/paddle_to_torch/rules.py:1]: 最好按照新版的rules来写

**Comment by cszdrg** [comment]
```
o1 = paddle.arange(1,3,-1)
print(o1)
```
paddle支持这种写法并输出空的tensor  但是torch不支持，需要修改config_analyzer初始化函数吗？
我感觉这不算初始化问题吧，应该放到torch不支持的文件里，你先提交 看看 @wanghuancoder导师怎么说

**Review by mzj104 — APPROVED** [review]
LGTM

**Review by wanghuancoder — APPROVED** [review]
LGTM
